src/tvgame/tvg_client.c

```diff
@@ -108,10 +108,23 @@ void SP_info_player_start(gentity_t *ent)
 /**
  * @brief QUAKED info_player_intermission (1 0 1) (-16 -16 -24) (16 16 32) AXIS ALLIED
  * The intermission will be viewed from this point.  Target an info_notnull for the view direction.
- *
- * @param ent - unused
  */
 void SP_info_player_intermission(gentity_t *ent)
+{
+	if (ent->spawnflags < 0 || ent->spawnflags > 2)
+	{
+		Com_Printf("SP_info_player_intermission: incorrect spawnflag %d\n", ent->spawnflags);
+		return;
+	}
```

Comment:
I haven't looked at how the spawn implementation works here so this might not be relevant, but I think this can potentially be wrong. Spawnflags are bitflags, so it's possible to have an `info_player_intermission` entity with `spawnflags 3`, meaning the intermission point is the same regardless of which team wins.

Reply:
Ah, you are most likely right because `FindIntermissionPoint` does bitwise operation on it but my brain didn't record it untill you pointed it out, thanks I will look into how handle this.